grovedb/Cargo.toml

```diff
@@ -7,21 +7,21 @@ edition = "2021"
 license = "MIT"
 
 [dependencies]
-rs_merkle = "1.1.0"
+rs_merkle = { version = "1.1.0", optional = true }
```

Comment:
Do we use rs-merkle btw? As I remember it was for a root tree and we changed that to Merk long ago

Reply:
Yeah, true think it was used in a test, will get rid of it.

Reply:
Done